_candidates_en/YoshiakiShibutani_1.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Yotchang
+title: "Let Scala handle it: Syntactic analysis of cafebabepy, a Python implementation on JVM"
+length: 40
+audience: Beginner
+language: Japanese
+twitter: https://twitter.com/yotchang4s
+github: https://github.com/yotchang4s
+icon: https://www.gravatar.com/avatar/dede7dcf77241349e1d130cd2363d858?s=200
+organization: F-CODE, Inc.
+tags:
+  - Tools
+  - Others
+suggestions:
+  - Those interested in language implementations
+  - Those interested in Python
+  - Those considering adopting the parser combinator in Scala
+---
+I am currently making a Python implementation on JVM called cafebabepy, using Java. Still work in progress, really. For the REPL (read–eval–print loop), I've let Scala's parser combinator handle the syntactic analysis of cafebabepy.
```

Comment:
Consider changing to "+I am currently making a Python implementation on JVM called cafebabepy, using Java. It's still a work in progress, really. For the REPL (read–eval–print loop), I've let Scala's parser combinator handle the syntactic analysis of cafebabepy."